src/v2/Components/v2/ArtworkFilter/ArtworkFilters/FollowedArtistsFilter.tsx

```diff
@@ -17,13 +18,13 @@ export const FollowedArtistsFilter: FC = () => {
     ),
   }
 
+  const count = get(filterContext, f => f.counts.followedArtists, 0)
```

Comment:
#nonblocking But could also do this with fancy typescript `f?.counts?.followedArtists ?? 0`!

Reply:
Ah, I may not have been using that properly!